tools/packaging/packaging.strings.psd1

```diff
@@ -127,7 +127,7 @@ open {0}
         <licenseUrl>https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt</licenseUrl>
         <tags>PowerShell</tags>
         <language>en-US</language>
-        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+        <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
```

Comment:
Why not "(c)"? We use in all files:
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Reply:
Agreed. Maybe we can change to use `(c)`?

Reply:
NuGet.org documentation needs it this way or the copyright symbol.

Reply:
agree, our legal teams says OSS should use (c).  I sent @adityapatwardhan 	 the link offline.

Reply:
I checked with nuget folks, and the only acceptable values are the copyright symbol and &#169;